### DISCUSSION TITLE
local coverart stops updating after using UPNP, stuck in upnp-mode

### DISCUSSION THREAD

**Discussion by kreativkod** [rern/rAudio#1512] [Bug Reports]
I use BubbleUPnP to stream Qobuz into rAudio. Whenever I stop that and select something from my NAS library instead, the music will start to play, but the coverart will not be updated (default cover shown) and the UPNP identification icons remain as well (in the bottom bar as well as in the status section). The image below shows music from my NAS playing although one would think it is via upnp.

The above goes for when I chose artist/composer etcand then go into and view the album with all the tracks and do replace and play from there.

If I stay on the page before, that lists composer/artist "albums",  and click the small thumbnail and do replace and play, the play page will flash briefly and then the playlist page will load without anything in it and no music begins to play.

So two bugs. Coverart stops working and playback stops working, depending on from where replace and play is executed.

![image](https://github.com/rern/rAudio/assets/21079205/e9151f3b-7b1a-4d48-853c-470a3fafb4ca)

**Comment by kreativkod** [comment]
A series of ctrl-F5 in Chrome yields this:
![image](https://github.com/rern/rAudio/assets/21079205/bd5cf9cf-18ff-4b73-bb4c-d31caf8affdc)

**Comment by kreativkod** [comment]
A potential work-around (I have not tested thoroughly) might be to actively press Stop in rAudio to cease all upnp activity, before trying to add music from the local library.

**Reply by rern** [reply]
It should be fixed by update 20231128.
